cmd/lookoutd/serve.go

```diff
@@ -7,23 +7,25 @@ import (
 	"github.com/src-d/lookout/server"
 	"github.com/src-d/lookout/util/cli"
 	"github.com/src-d/lookout/util/ctxlog"
+
+	gocli "gopkg.in/src-d/go-cli.v0"
 )
 
 func init() {
-	if _, err := app.AddCommand("serve", "run a standalone server", "",
-		&ServeCommand{}); err != nil {
-		panic(err)
-	}
+	app.AddCommand(&ServeCommand{})
 }
 
 type ServeCommand struct {
+	gocli.PlainCommand `name:"serve" short-description:"run a standalone server" long-description:"Run a standalone server"`
 	queueConsumerCommand
 }
 
 func (c *ServeCommand) Execute(args []string) error {
 	ctx, stopCtx := context.WithCancel(context.Background())
 	stopCh := make(chan error, 1)
 
+	cli.InitLog(&c.LogOptions, name)
```

Comment:
What do you think if we use `cli.Command` instead of `cli.PlainCommand`? It would bring profiling by default now and any other perks in the future. Also I really like that `go-cli` calls init by default. Right now we don't use it.

My suggestion:
```
type LookoutCommand struct {
  cli.Command
}

func (c LookoutCommand) Init(a *App) error {
  c.Command.Init()
  // add additional logic for our logger
}
```

Reply:
I'd prefer to leave the full `cli.Command` out of this PR to limit it to migrate what we have to go-cli, as is. We can add profiling later easily if we think we need it.

For the initializer interface, I added it in 2812a43

Reply:
makes sense 👍